portfolio/.classpath

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
.classpath and all are not necessary.
Please create a gitignore file and all system folders to github

Reply:
You can refer this for a comprehensive list of items in gitignore
https://gist.github.com/jacks0n/cc0386e71d30c66a670e